08LessonEight/in-class-project/homework.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+    <title> Blog Home Page </title>
+    <link rel="stylesheet" href="css/homework-style.css" />
+</head>
+
+<body>
+    <div id="container">
+        <header>
+            <h1> Recent From Blog </h1>
+            <p>Donec tincidunt felis quam, eu tempus purus finibus in. Curabitut henderit, odio in viverra interdum, lorem velit scelerisque ipsum, a sagittis ligula leo in dolor. Etiam vestibulum.</p>
+        </header>
+        <main class="clearfix">
+            <section>
+                <div class="leftbox">
+                    <img src="img/keyboard.jpg" width="345px" height="150px">
+                    <h2>Lorem ipsum dolor sit amet. </h2>
+                    <p> Phasellus faucibus odio vitae iaculis lobortis. Ut bibendum tincidunt justo. Duis ultricies, mauris in. </p>
+                    <button> Read More </button>
+                </div>
+            </section>
+
+            <section id="clearfix" class="rightbox">
+                <div id="clearfix" class="headphones">
+                    <img src="img/laptop.jpg">
+                    <h3> Suspendisse a sagittis </h3>
+                    <p> Sed ut sem elit. Pelientesque vitae erat vehicula quam posuere porta. </p>
+                </div>
+                <div id="clearfix" class="laptop">
+                    <img src="img/desk.jpg">
+                    <h3> Morbi eu metus ut libero </h3>
+                    <p> Pelientesque magna dolor, dapibus vitae lorem in, laoreet sagittis lacus. </p>
+                </div>
+                <div id="clearfix" class="window">
+                    <img class="textwrap" src="img/window.jpg">
+                    <h3> Pellentesque magna dolor </h3>
+                    <p> Suspendisse a sagittis tortor, ac dapibus turpis. </p>
+                </div>
+            </section>
+
+
+            <section id="clearfix" class="bottomcontainer">
+                <div class="bottomleft">
+                    <h2> Why profession? </h2>
+                    <ul> &#x2714 Clean & Corporate Webdesign </ul>
```

Comment:
Remember to use `&amp;` for the `&` symbol in your text elements. Also, `&#x2714` should have a semi-colon (`&#x2714;`)